### ISSUE TITLE
Bump to 2.6.1

### ISSUE THREAD

**Issue by Drenmi** [whitequark/parser#553]
A [new version was released](https://www.ruby-lang.org/en/news/2019/01/30/ruby-2-6-1-released/) about three weeks ago. I have read through the [commit log](https://github.com/ruby/ruby/compare/v2_6_0...v2_6_1), and it shouldn't contain any updates to the parser. Can we please bump `parser` version to remove:

```
warning: 2.6.0-compliant syntax, but you are running 2.6.1.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
```

**Comment by douglas** [comment]
This would be amazing as it is the only thing preventing a upgrade to 2.6.1 for an app that I maintain - do you need any help?

**Comment by iliabylich** [comment]
It's just a warning and @Drenmi is right, there were no changes in grammar in 2.6.1.  You can safely use `2.6.0.0`.

**Comment by douglas** [comment]
Sure! It works, but we're just avoiding push the upgrade because of the warning as it shows up every time we run the tests.

Just to be clear, we're not in a rush, tho! Just wanted to offer help if needed.

**Comment by iliabylich** [comment]
@whitequark Could you release it please?

**Comment by Drenmi** [comment]
Would be great to have this released, as everyone running RuboCop with Ruby 2.6.1 keeps getting this warning. 🙇

**Comment by bbatsov** [comment]
Btw, 2.6.2 is also out. I think it doesn't have AST related changes.
